corehq/apps/hqcase/api/get_list.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+from itertools import chain
+
+from dateutil.parser import parse
+
+from dimagi.utils.parsing import FALSE_STRINGS
+
+from corehq.apps.es import case_search
+from corehq.apps.es import cases as case_es
+
+from .core import UserError, serialize_es_case
+
+DEFAULT_PAGE_SIZE = 20
+MAX_PAGE_SIZE = 5000
+CUSTOM_PROPERTY_PREFIX = 'property.'
+
+
+def _to_boolean(val):
+    return val.lower() not in [''] + list(FALSE_STRINGS)
+
+
+def _to_int(val, param_name):
+    try:
+        return int(val)
+    except ValueError:
```

Comment:
Is `val` always expected to be a string? If not (if it could be `None`, for example), this should also catch `TypeError`

Example error with `val = None`
```
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
```suggestion
    except (ValueError, TypeError):
```

Reply:
So long as this is an HTTP API, yes this should always be a string, as there's no type information in a querystring, and Django's QueryDict doesn't attempt duck typed coercion.  I could see that changing if we were to open this up to a different interface, but I think it's reasonable to hold off on that sort of thing for now, as I'd want stricter type controls in such a scenario anyways (including for the string values).

FWIW:
```python
>>> QueryDict('foo=&bar=baz').get('foo')
''
```